animations/index.md

```diff
@@ -1,159 +1,88 @@
+For more information, see [Transition animations](/animations/transitions_widgets.html).  
+
+
+### Explicit animation widgets
+
+ *Explicit animations* involve building complex animations using the `AnimationBuilder` widget which requires you to customize many of the animation elements that are pre-defined in the implicit and transition animation widgets. When you build explicit animations, you manually add listeners, tickers, and other elements for customization.  
+ For more information, see [Explicit animations](/animations/explicit_widgets.html).  
+
+
+## Flutter packages
+The Flutter SDK includes several libraries that contain pre-defined animation widgets. To access the Flutter animation widgets, import the Flutter package that contains the animation widget that you want to use.   Determine which library contains the widget and then add the appropriate import statement to the `main.dart` file.
+
+```Dart
+import 'package:flutter/animation.dart';
+import 'package:flutter/material.dart';
+```
+For information on the SDK libraries and available widgets in each library, see the following resources.
+
+* [Flutter SDK libraries](https://docs.flutter.io/index.html)  
+* [Animation and Motion Widgets](/widgets/animation/)  
+* [Flutter Widget Index](/widgets/widgetindex/)  
+* [Widget Catalog](/widgets/)
+
+## Flutter animation resources  
+To learn about Flutter animations, check out the following resources.
+
+### Concepts  
+For information about how Flutter animation works and definitions of the main animation concepts see [Flutter Animation Concepts](/animations/concepts.html). You'll also find information here on the main  animation widgets such as the `AnimationController` or `AnimationBuilder`, and information about tweens, tickers, and more.  
+
+### Animation and Motion widgets  
+For a list of Flutter SDK animation widgets, see the following:
+* [Implicit animation widgets](/animations/implicit_widgets.html)  
+* [Transition animation widgets](/animations/transitions_widgets.html)  
+* [Explicit animation widgets](/animations/explicit_widgets.html)  
+* [Animation and Motion Widgets Catalog](/widgets/animation/)  
+
+### Codelabs  
+Codelabs are a great way to learn how to use Flutter and how to add Flutter animation widgets to your app. Check out [Building Beautiful UIs with Flutter](/codelabs/). This Codelab demonstrates how to build a simple chat app. And Step 7 (Animate your app) shows how to animate the new message—sliding it from the input area up to the message list. For more information, see [Codelabs](/codelabs/).  
+If you're new to Flutter, check out the [Write your first app, part 1](https://codelabs.developers.google.com/codelabs/first-flutter-app-pt1/#0) and [Write your first app, part 2](https://codelabs.developers.google.com/codelabs/first-flutter-app-pt2/#0) codelabs. They walk you through the basics of using Flutter to quickly create an iOS and Android app.  
+
+### Flutter animation demos and examples  
```

Comment:
The items below are not really "demos and examples," except the AnimatedList sample. In the current Animations landing page, we call them "Common animation patterns". Which I think is still a useful notion.